configs/components/oifs/oifs.yaml

```
@@ -761,6 +780,27 @@ choose_wam:
 # OTHER CONFIGS AND SWITCHES
 #======================================================================================
 
+# For coupled setups we can use ocean currents
+choose_use_ocean_currents:
+        1:
+                add_namelist_changes:
+                        fort.4:
+                                NAEPHY:
+                                        LECURR: ".true."
+        0:
+                add_namelist_changes:
+                        fort.4:
+                                NAEPHY:
+                                        LECURR: ".false."
```

This makes sure we use a boolean here on resolution of the `choose`.
```suggestion
        True:
                add_namelist_changes:
                        fort.4:
                                NAEPHY:
                                        LECURR: ".true."
        False:
                add_namelist_changes:
                        fort.4:
                                NAEPHY:
                                        LECURR: ".false."
```

---

@JanStreffing, did you deleted this on the past? Any comments?